lib/v4/MessageBuilder.ts

```diff
@@ -0,0 +1,75 @@
+import { type NonEmptyArray } from '../utils/NonEmptyArray.ts';
+import { defaultErrorMapOptions } from './errorMap/index.ts';
+import type * as zod from 'zod/v4/core';
+
+export type ZodIssue = zod.$ZodIssue;
+
+export type MessageBuilder = (issues: NonEmptyArray<ZodIssue>) => string;
+
+const stubErrorMap: zod.$ZodErrorMap<zod.$ZodIssue> = (issue) => {
```

Comment:
suggestion(nit): assuming `stub` or `mock` terms are most commonly used in the context of test files 

```suggestion
const identityErrorMap: zod.$ZodErrorMap<zod.$ZodIssue> = (issue) => {
```

Reply:
14927ba